documentation/src/main/asciidoc/reference/mapper-orm-indexing-massindexer.asciidoc

```diff
@@ -77,6 +77,30 @@ include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/indexing/Hibe
 <2> Start the mass indexing process for the selected types and return when it is over.
 ====
 
+You can also select a subset of target entities to be reindexed.
+
+.Partial reindexing
```

Comment:
I'd argue that "partial" is a bit confusing, since the example above, "Reindexing selected types using a `MassIndexer`", can also be considered as "partial" reindexing.

Maybe rather "conditional" reindexing?

Reply:
We already use the phrase `Partial reindexing` for documenting the similar feature for the batch JSR-352.
I'm going to change it for both.